src/fix-libraries.js

```diff
@@ -44,7 +45,7 @@ function updateProject (project) {
 
 					// Copy that bad boy.
 					const clone = JSON.parse(JSON.stringify(sourceConfig));
-					clone.name = destinationBuildConfig;
+					clone.name =  util.format('"%s"', destinationBuildConfig);
```

Comment:
Nope, I'm just tired, haha. 

This should just be: 

``clone.name =  `"${destinationBuildConfig)}"`;``

I'll update it now.